starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -49,4 +49,27 @@ public void findAll() {
     List<Inventory> actualInventory = this.inventoryDAO.findAll();
     Assert.assertFalse(actualInventory.isEmpty());
   }
+  
+  /**
+   * Test Create method 
+   */
+  @Test
+  public void create() {
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
+    inventory.setProductType(PRODUCT_TYPE);
+    Inventory actualInventoryItem = this.inventoryDAO.create(inventory);
+    Assert.assertEquals(PRODUCT_TYPE, actualInventoryItem.getProductType());
+    Assert.assertEquals(NAME, actualInventoryItem.getName());
+    
+    // Test for more than one inventory 
```

Comment:
We don't need to test inventory size as we want to limit our unit tests to the method we're testing.
Anything below line 65 can be deleted.

Reply:
Lines removed in commit beee9ac62fec7cdbca505d54a12b1681a8a629eb